crates/matrix-sdk-ui/src/sync_service.rs

```diff
@@ -321,6 +321,11 @@ struct SyncServiceInner {
 /// with a Matrix server. It can initiate and maintain the necessary
 /// synchronization tasks for you.
 ///
+/// **Note**: The [`SyncService`] requires as server with support for [MSC4186],
```

Comment:
nit: a server, not as server

Reply:
07f86de2919aaf6158aff4b2f9d32e73f3d76cea